src/components/schemas/expense.yaml

```diff
@@ -1678,29 +1678,81 @@ expense_delete_in:
 
 
 expenses_stats_in:
+  type: object
+  required:
+    - data
   properties:
     query_params:
-      nullable: False
+      nullable: false
       type: string
       description: | 
-        This field contains the query param string similar to the query params passed in expenses GET call for displaying expenses in the list view.
-      example: id=in.('tx1263', 'tx1569')
+        Query param string similar to expenses GET call. Use group_by_state=eq.true to get state-wise grouping of stats.
+      example: id=in.('tx1263', 'tx1569')&group_by_state=eq.true
 
 expenses_stats_out:
-  type: object 
-  properties:
-    count:
-      type: integer
-      nullable: False
-      description: |
-        This field represents the total number of expenses that match the criteria specified in the query_params within the body of the request.
-      example: 1     
-    total_amount:
-      allOf:
-        - $ref: './fields.yaml#/amount'
-      description: |
-        This field indicates the sum of amount of expenses that match the criteria specified in the query_params within the body of the request.
-      example: 100
+  type: array
```

Comment:
Shouldn't this be `OneOf`? Case1: not group by and Case2: group by?

Reply:
Updated